flexport/latitude
# Move ItemRenderer out of Table function

This branch builds off of https://github.com/flexport/latitude/pull/95. It refactors the ItemRenderer out of the Table function to prevent it from remounting on every table re-render.

Manual test plan:
Run storybook and go to the Rerender Table Test.
Add `React.useEffect(() => console.log('item renderer mount'), [])` to the ItemRenderer function and see that it does not remount on every table re-render.
